File: src/main/java/com/stacked/sigaa_ifc/Aula.java
package com.stacked.sigaa_ifc;

import java.util.ArrayList;

public class Aula {
    private String titulo, html;
    private ArrayList<Anexo> anexos = new ArrayList<>();
    private Disciplina disciplina;

    Aula(Disciplina disciplina, String titulo, String html) {
        this.disciplina = disciplina;
        this.titulo = titulo;
        this.html = html;
    }

    //anexosSIAGA

    public String getTitulo() { return titulo; }
    public String getHtml() { return html; }
    Disciplina getDisciplina() {return disciplina;}

    void adicionarAnexo(Anexo a) { anexos.add(a); }
    public ArrayList<Anexo> getAnexos() { return anexos; }

}

File: src/main/java/com/imawa/sigaaforkotlin/models/Aula.kt
package com.imawa.sigaaforkotlin.models

/**
 * Representa uma aula cadastrada em uma disciplina.
 */
data class Aula(
    val titulo: String,
    val htmlConteudo: String,
    val anexos: ArrayList<Anexo>,
    val disciplina: Disciplina
)
